app/components/shared/process_steps_component.rb

```diff
@@ -6,13 +6,27 @@ def initialize(process:, service:, title:)
   end
 
   def render?
-    @process.blank? || %w[create review].include?(@process.state)
+    @process.blank? || %w[create draft review].include?(process_state)
```

Comment:
I think the abstract way to do this would be to make the process classes (Vacancy and JobApplication) responsible for knowing what states are renderable: it'll be a different list for every possible process. E.g.:

```ruby
class IceCreamOrder
  def states_where_the_process_steps_component_should_render
    %w[in_queue choosing_toppings adding_sprinkles]
  end
end
```